db/changelog/db.changelog-main.yml

```diff
@@ -0,0 +1,3 @@
+databaseChangeLog:
```

Comment:
Any specific reason for the main changelog file and the nested one not having the same format (YAML vs JSON)? If not, I think we should use the same format for all changelog files

Reply:
Based on the discussion we had with Stephen, I believe we decided on yaml for the main changelog as the format kept it a bit more concise for simply routing to other files.
Then we decided on json for the actual changelog files as it structured that sort of meta data a bit nicer.